phylesystem_api/phylesystem_api/views/default.py

```diff
@@ -180,8 +179,8 @@ def fetch_and_cache(url):
                 ct = "text/plain"
             return Response(
                 headers=fetched.headers,
-                body=fetched.text,  # missing JSON payload will raise an error
-                status="200 OK",
+                body=fetched.text,
+                status=fetched.status_code,  # keep the returned status code!
```

Comment:
Yes, I see that  `pyramid.response.Response` has three(!) status properties, which could easily get out of sync:
 - `status` (descriptive string, starting with status code)
 - `status_code` (as number)
 - `status_int`  (as integer)

Out of curiosity, I tried changing each of these in a local debugger, and in all cases they stay in sync:
```python
(Pdb) test
<Response at 0x1069f00d0 200 OK>
(Pdb) test.status
'200 OK'
(Pdb) test.status_code
200
(Pdb) test.status_int
200
```

Now let's assert a bogus status code+string:
```python
(Pdb) test.status = "499 fooBAR"
(Pdb) test.status
'499 fooBAR'
(Pdb) test.status_code
499
(Pdb) test.status_int
499
```
And let's try some known status codes (no string provided):
```
(Pdb) test.status_code = 403
(Pdb) test.status
'403 Forbidden'
(Pdb) test.status_int
403
(Pdb) test.status = 301
(Pdb) test.status
'301 Moved Permanently'
(Pdb) test.status_int
301
(Pdb) test.status_code
301
```
So it's pretty robust! But yes, for consistency I'll do as you suggest above. Since this is a known status code, the correct string will be assigned automatically.

Reply:
NB These changes are now live on **dev**, if you want to kick the tires.